lib/parse-mvn.js

```diff
@@ -77,31 +77,42 @@ function assemblePackage(source, projectDeps, parent, withDev) {
 
 function createPackage(pkgStr, parent) {
   var range = getConstraint(pkgStr);
+
   if (range) {
     pkgStr = pkgStr.substring(0, pkgStr.indexOf(' '));
   }
+
   var parts = pkgStr.split(':');
+  var version = parts[3];
+  var groupId = parts[0];
+  var artifactId = parts[1];
   var result = {
-    groupId: parts[0],
-    artifactId: parts[1],
+    groupId: groupId,
+    artifactId: artifactId,
     packaging: parts[2],
-    version: parts[3],
+    version: version,
     name: parts[0] + ':' + parts[1],
     dependencies: {},
   };
```

Comment:
You can leave this part as is, without defining the separate vars outside of `result`, no? Seems cleaner, even if you overwrite the version in some cases later on.